search/includes/classes/class-search.php

```diff
@@ -117,6 +117,16 @@ public function init() {
 		$this->setup_regular_stat_collection();
 	}
 
+	/**
+	 * Check if the constants needed for ElasticSearch connection are defined.
+	 */
```

Comment:
Should we add the return type on the PHPDoc?

Reply:
Sure, done.